.github/actions/reportportal-summarize/get-teams-message.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+if [[ -n "$RP_LAUNCH_KEY" ]]; then
+
+  NB=$(echo "$RP_CONTENT" | jq -r '.page.totalElements // "0"')
+  if [[ "$NB" == "0" || -z "$NB" ]]; then
+    MSG+="No report found for key "'`'"$RP_LAUNCH_KEY"'`'"."
+    MSG+="\n\nSee [latest reports]($RP_LAUNCH_URL)."
+  elif [ "$NB" == "1" ]; then
+    RP_LAUNCH_ID=$(echo "$RP_CONTENT" | jq -r '.content[0].id // empty')
+    STATUS=$(echo "$RP_CONTENT" | jq -r '.content[0].status // empty')
+    [[ "$STATUS" == 'PASSED' ]] && ICON="✅" || ICON="❌"
+    MSG+="See [report]($RP_LAUNCH_URL/$RP_LAUNCH_ID) $ICON"
+  else
+    MSG+="$NB reports found for key "'`'"$RP_LAUNCH_KEY"'`'"."
+    while read -r id ; do
+      read -r number
+      read -r status
+      case $status in
+        PASSED)
+        sstatus="✅" ;;
```

Comment:
Maybe the variables deservers some renaming, I'm bit lost between `STATUS` (capital), `status` (lower case) and `sstatus`. What does the `s` prefix stands for in the latest one? String?

Reply:
I updated to make it clearer, choice of variables was linked to a battle with bash logic